src/components/createEditCqlLibrary/CreateEditCqlLibrary.tsx

```diff
@@ -116,81 +117,99 @@ const CreateEditCqlLibrary = () => {
   }
 
   return (
-    <div tw="m-5">
-      {serverError && (
-        <ErrorAlert data-testid="cql-library-server-error-alerts" role="alert">
-          {serverError}
-        </ErrorAlert>
-      )}
-      <form
-        data-testid="create-new-cql-library-form"
-        onSubmit={formik.handleSubmit}
+    <>
+      <div
+        tw="flex flex-wrap "
+        className="flex justify-center md:justify-end"
```

Comment:
looks like we are adding the classes twice

Reply:
Thanks, I'll remove it